smurfs/src/components/Smurfs.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { getSmurfs, addSmurf } from '../actions';
+import SmurfCards from './SmurfCards';
+
+const Smurfs = props => {
+  const [smurf, setSmurf] = useState({name: '', age: '', height: ''});
+
+  const handleChanges = e => {
+    setSmurf({...smurf, [e.target.name]:e.target.value})
+  }
+
+  useEffect(() => {
+    props.getSmurfs();
+  }, [])
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    props.addSmurf(smurf);
+    setSmurf({name: '', age: '', height: ''})
+  }
+
+  if (props.Fetching){
+    return <h2>GATHERING SMURFS...</h2>
+  }
+
+  return(
+    <div>
+      {props.error && <p>{props.error}</p>}
+      {props.smurfs.map(item => (
+        <SmurfCards key={item.id} smurf={item}/>
+      ))}
```

Comment:
This content would fit best in a separate component - that way, we have a single form component that we can reuse as much as we need 👍